controllers/admin/AdminDashboardController.php

```diff
@@ -304,7 +304,7 @@ public function renderView()
             'hookDashboardZoneTwo' => Hook::exec('dashboardZoneTwo', $params),
             'action' => '#',
             'warning' => $this->getWarningDomainName(),
-            'new_version_url' => Tools::getCurrentUrlProtocolPrefix() . _PS_API_DOMAIN_ . '/version/check_version.php?v=' . _PS_VERSION_ . '&lang=' . $this->context->language->iso_code . '&autoupgrade=' . (int) ($moduleManager->isInstalled('autoupgrade') && $moduleManager->isEnabled('autoupgrade')) . '&hosted_mode=' . (int) defined('_PS_HOST_MODE_'),
+            'new_version_url' => _PS_API_URL_ . '/version/check_version.php?v=' . _PS_VERSION_ . '&lang=' . $this->context->language->iso_code . '&autoupgrade=' . (int) ($moduleManager->isInstalled('autoupgrade') && $moduleManager->isEnabled('autoupgrade')) . '&hosted_mode=' . (int) defined('_PS_HOST_MODE_'),
```

Comment:
Using `Tools::getCurrentUrlProtocolPrefix` was wrong here, for `api.prestashop.com` https about be always used.